code/modules/reagents/chemistry/reagents/pyrotechnic.dm

```diff
@@ -224,8 +224,9 @@
 /datum/reagent/cryostylane/on_tick()
 	if(holder.has_reagent("oxygen"))
 		holder.remove_reagent("oxygen", 1)
-		holder.chem_temp -= 10
-		holder.handle_reactions()
+		holder.remove_reagent("cryostylane", 1)
+		holder.temperature_reagents(holder.chem_temp - 200)
+		holder.temperature_reagents(holder.chem_temp - 200)
```

Comment:
Goon has a system whereby chemistry reactions can be delayed, so they occur slowly, as opposed to instantly. 

I'm not going to implement that system in this PR, but I probably will, eventually, because `on_tick` is kinda a gross system, and having every single reagent datum be in object processing is equally as disgusting.

Short run of their system: if a reaction is delayed, then it gets added to a list and it runs part of the reaction, once per second, until it runs out.

This also means that cryostylane and pyrosium's temperature interactions could occur inside mobs. 

But that's for another PR.